src/app/profile/settings/resources/resources.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="container-fluid container-cards-pf">
+  <div class="col-xs-6">
+    <div class="row resource-header-colour list-group-item">
+      <div>
+        <div class="col-sm-6">
```

Comment:
If you are using `col-xs-` as the parent column definition, please continue that into the child elements (change `col-sm-` to `col-xs-`).